src/object-identifier.scss

```diff
@@ -0,0 +1,61 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-object-identifier;
+
+.#{$block} {
+  @mixin fd-object-identifier-title-base {
+    @include fd-reset();
+
+    color: var(--sapList_TextColor);
+  }
```

Comment:
You don't need a mixin for code that is used only once.

Reply:
fixed